pkg/agent/flowexporter/connections/interface.go

```diff
@@ -23,3 +23,10 @@ import (
 type ConnTrackDumper interface {
 	DumpFlows(zoneFilter uint16) ([]*flowexporter.Connection, error)
 }
+
+// ConnTrackInterfacer is an interface created to consume the required dump functions from either the third party
+// conntrack library or internal packages depending on OVS datapath type or OS.
+type ConnTrackInterfacer interface {
+	GetConnTrack(config interface{}) error // suggest a different name for config if it is not appropriate
```

Comment:
I'm starting too feel we have too much abstraction and too many interfaces here. But this method is definitely the most confusing IMO. For one, it's called "Get*" but it does not return anything...

I don't have a good understanding of what's going on anymore, so it's hard for me to suggest an alternative, but it's pretty clear to me that this interface needs to be rethought. An interface where all the parameters are themselves arbitrary objects (`interface{}`), whose actual type depends on the specific interface implementation, is not a helpful abstraction